app/assets/javascripts/stores/questionnaire_store.cjsx

```diff
@@ -53,4 +55,19 @@ class QuestionnaireStore extends EventEmitter
   addChangeListener: (callback) =>
     @on(EVENT, callback)
 
+  saveAll: =>
+    token = $('meta[name="csrf-token"]').attr('content')
```

Comment:
You could replace this with `document.getElementsByName("csrf-token")[0].content`, and spare requiring jquery. Fewer KBs for us! 😀